src/code/code_800EC960.c

```diff
@@ -3040,9 +3040,9 @@ void AudioDebug_Draw(GfxPrint* printer) {
             GfxPrint_Printf(printer, "DEMO FLAG %d", sAudioCutsceneFlag);
 
             GfxPrint_SetPos(printer, 3, 12);
-            if (sAudioHasMalonBgm == true) {
-                GfxPrint_Printf(printer, "MARON BGM DIST %f", sAudioMalonBgmDist);
-                sAudioHasMalonBgm = false;
+            if (sAudioDebugIsMalonSingingUpdating == true) {
+                GfxPrint_Printf(printer, "MARON BGM DIST %f", sAudioDebugMalonSingingDist);
+                sAudioDebugIsMalonSingingUpdating = false;
```

Comment:
these variable names are kind of a mouthful. could IsMalonSinging work on its own? 

and also does it need debug in the name? it does get used for a debug print here but it could theoretically be used as a general check that malon is singing within the audio system if desired right? may or may not be a good idea, not sure

Reply:
The idea was to have variables intended for debug start with the `sAudioDebug` so it's clear reading the code what's the main code and what's the debug code (mainly this comes up in a different file).

For something simple like this, yeah it doesn't necessarily need the `AudioDebug` part. So I'd be fine with `sIsMalonSinging` and `sMalonSingingDist`